content/prototype-your-solution/index.md

```diff
@@ -0,0 +1,29 @@
+---
+title: Prototype Your Solution
+content_type: project
+submission_type: link 
+---
+
+Yesterday you came up with an actionable plan and created a storyboard for your prototype. Today, you’ll adopt a “fake it” philosophy to turn that storyboard into a prototype. A realistic façade is all you need to test with users. Prototyping is getting ideas and explorations out of your head and into the physical world. A prototype can be anything that takes a physical (or digital) form that a user can interact with. **Note:** This step can become a bottleneck between dream and reality, but often the simple prototype is the most effective  -  don’t worry about it being fully operational, lead with a basic mock-up or rendering.

```

Comment:
Yesterday . Consider rewording. Remember that this is self paced. What if they did the work this morning?